010-Getting-started/010-what-is-xata.mdx

```diff
@@ -1,300 +1,131 @@
+Xata achieves cost efficiency for as many use cases as possible is by offering a high degree of optionality. The innovation of the Xata service is that we make it really easy to move from one type of cluster to another, without downtime. This means that you can easily try your workload on different types of clusters and see which one is the most cost-efficient for you.
 
-If you just want to build apps and use the Xata serverless service, you **do not** need to read this document. We’re solving these problems so that you don’t have to.
+At small scale, we offer a free tier that includes high-availability and read replicas, no cold-starts, and no instance-pausing after a few days. We can offer this sustainably because the free tier makes use of [shared clusters](#shared-or-dedicated-clusters) and auto-scaling.
 
-Xata is a global multi-tenant system, composed of **cells**, each living in a region. At the high level, one Xata **cell** implements the following architecture:
+At large scale, you pay for the resources you use and you have a high degree of control. For example, for predictable load, you can configure a particular instance size, while for unpredictable load, auto-scaling will likely be more cost-efficient. Similarly, you can choose between standard storage and I/O optimized storage, which is more cost-efficient for I/O-heavy workloads.
 
-![Xata cell](/images/xata-cell.png)
+### Scalability
 
-- The data is stored in PostgreSQL, which is considered the source of truth.
-- The PostgreSQL logical replication events are written in a Kafka queue, from where they can be read by multiple consumers.
-- One of these consumers optionally indexes the data into Elasticsearch.
-- The Xata API layer offers a RESTful API offering both the functionality of the relational database (PostgreSQL) with the advanced search and analytics from the search engine/columnar store (Elasticsearch).
-- On top of this API, Xata has client SDKs and a web interface.
+Available instance types in Xata go up to 128 vCPUs and 1TB of RAM (for example, `db.r6i.32xlarge`). Storage can be scaled up to 128TiB per cluster. That is quite a lot of headroom for vertical scaling.
 
-While the above architecture is a very common one in the industry, it hasn’t been —until now— productized in a service, which means a lot of teams were forced to either implement something similar to the above or live with reduced functionality.
+When it comes to horizontal scaling, the fact that you can create databases and branches instantly and that you can move them between clusters without downtime, opens up several architectural possibilities. For example, you can create a logical database for each of your customers, and place them in different regions. See the section on [Global control plane for databases and branches](#global-control-plane-for-databases-and-branches) for more details.
 
-## Going global
+### Infrastructure optionality
```

Comment:
@tsg This is the only wording change I made that you might not like. This previously said "shared and dedicated clusters", which are already the headings below this one. I liked how you used the word "optionality" when describing this strategy so I just bubbled it up a bit.

Reply:
I like it 👍